app/test/transformers/webpacker.js

```
@@ -0,0 +1,31 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require("fs");
+const tsjest = require("ts-jest");
+
+const transformer = tsjest.createTransformer();
+
+// look for `?raw` import statements
+// re-write these into `const variable = "string source";`;
+const importRegEx = /^import (.*) from "(.*)\?raw";$/gm;
```

Nit: might want to validate that p1 is a single identifier. We wouldn't support `import {a,b} from "...?raw"` and should probably throw an error in that case.